pkg/cmd/logs.go

```diff
@@ -535,23 +540,23 @@ func logsTerraform(toMatch string) {
 }
 
 // logsTf prints the logfiles of tf job
-func logsTf() {
-	logsTerraform("tf-job")
+func logsTfHelp() {
+	fmt.Println("Command must be in the format: logs tf (infra|dns|ingress) shoot mame")
 }
 
 // logsInfra prints the logfiles of tf infra job
-func logsInfra() {
-	logsTerraform("infra.tf-job")
+func logsInfra(string string) {
```

Comment:
Please name parameter `str string` to avoid confusion with keyword. Same for other occurences

Reply:
fixed